### ISSUE TITLE
new deprecation: `create-url-argument` *always* throws when an object is passed to create.

### ISSUE THREAD

**Issue by NullVoxPopuli** [san650/ember-cli-page-object#544]
```ts
// usage:
create({
  // this can be empty
});
// ...
function create(definitionOrUrl, definitionOrOptions, optionsOrNothing) {
    let definition;
    let url;
    let options;

    /////////////////////////
    // definitionOrUrl is an object
    if (typeof definitionOrUrl === 'string') {
      url = definitionOrUrl;
      definition = definitionOrOptions || {};
      options = optionsOrNothing || {};
    } else {
      ///////////////////////
      // so url is false
      url = false;
      definition = definitionOrUrl || {};
      options = definitionOrOptions || {};
    }

    let { context } = definition;
    definition = (0, _helpers.isPageObject)(definition) ? (0, _helpers.assign)({}, (0, _helpers.getPageObjectDefinition)(definition)) : assignDescriptors({}, definition);

    delete definition.context;

     //////////////////////////
     // url is false, so this deprecation is thrown
    Ember.deprecate('Passing an URL argument to `create()` is deprecated', typeof url !== 'string', {
      id: 'ember-cli-page-object.create-url-argument',
      until: "2.0.0",
      url: 'https://ember-cli-page-object.js.org/docs/v1.17.x/deprecations/#create-url-argument'
    });
```

**Comment by NullVoxPopuli** [comment]
I'd PR a fix, but my system is out of file watchers and won't start up ember-cli-page-object :thinking:

**Comment by ro0gr** [comment]
Ah, it seems to be reproduction for the  https://github.com/san650/ember-cli-page-object/pull/534. Thanks for figuring it out! A PR would be very welcome!

Regarding the watchers issue, I haven't noticed it ever before. I can only guess here. Do you do fresh install over the cloned project, w/o any linking?

**Comment by NullVoxPopuli** [comment]
it actually looks like on `master`, this is already fixed? 
https://github.com/san650/ember-cli-page-object/blob/master/addon-test-support/create.js#L213
the condition is inverted, but also the code entirely looks a bit different. :thinking: so.. idk what's going on yet.
I now wonder if I'm having yarn-workspace problems and I'm not actually using the latest ember-cli-page-object. O.o

yeah, so this was fixed here: https://github.com/san650/ember-cli-page-object/commit/6dbdb9e11210b6d2e9a2c94fe74f883802f0fdb0#diff-7e9135d873a4889a025f4ed419cf7d3025a1fc38f7ec08bb1c7b3800c55a3483

which is included in 1.17.7.... ugh. yarn

**Comment by ro0gr** [comment]
Wow, I didn't realize that refactoring does also fix this issue 😃 great to know!
